source/_integrations/enphase_envoy.markdown

```
@@ -83,111 +90,467 @@ This integration follows standard integration removal. No extra steps are requir
+#### Current transformer entities
+
+CT measure multiple properties of the energy exchange which are available as Envoy device entities. These are all disabled by default, enable them as desired.
+
+##### Production CT sensor entities
+
+- **Envoy <abbr title="Envoy serial number">SN</abbr> Frequency production CT**: Frequency in Hz.
+- **Envoy <abbr title="Envoy serial number">SN</abbr> Voltage production CT**: Voltage in V. (see limitations [Summed voltage](#summed-voltage))
+- **Envoy <abbr title="Envoy serial number">SN</abbr> Production CT current**: Current in A.
+- **Envoy <abbr title="Envoy serial number">SN</abbr> Powerfactor production CT**: Powerfactor, ratio of active to apparent power.
+- **Envoy <abbr title="Envoy serial number">SN</abbr> Metering status production CT**: Status of the metering process: `normal`, `not-metering`, `check-wiring`.
+- **Envoy <abbr title="Envoy serial number">SN</abbr> Meter status flags active production CT**: Count of CT status flags active. See troubleshooting [CT Active flag count](#ct-active-flag-count) when non-zero.
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)